Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -554,6 +554,20 @@ AsyncClipboardAPIEnabled:
     WebCore:
       default: false
 
+AsyncDocumentLifecycleEventsEnabled:
```

Comment:
It's probably confusing to use the term "lifecycle" in this feature flag since we have: https://wicg.github.io/page-lifecycle/spec.html